src/validation.h

```diff
@@ -407,6 +407,12 @@ class BlockManager {
     /** Create a new block index entry for a given block hash */
     CBlockIndex* InsertBlockIndex(const uint256& hash) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 
+    //! Mark one block file as pruned (modify associated database entries)
+    void PruneOneBlockFile(const int fileNumber) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+
+    // See definition for documentation
```

Comment:
Move the doxygen comments here (this comment made sense when it was next to a forward declaration, but now that the declaration is in the header file, it should have the doxygen comment)

Reply:
Resolved in cdd2063bbe8b8df4db42c81ee7913e2dc0b2387a